service/src/main/java/bio/terra/workspace/service/resource/controlled/flight/clone/workspace/AwaitCreateReferenceResourceFlightStep.java

```diff
@@ -59,7 +60,8 @@ public StepResult doStep(FlightContext context) throws InterruptedException, Ret
         FlightUtils.validateRequiredEntries(
             context.getWorkingMap(), ControlledResourceKeys.DESTINATION_REFERENCED_RESOURCE);
         final FlightState subflightState =
-            context.getStairway().waitForFlight(flightId, FLIGHT_POLL_SECONDS, FLIGHT_POLL_CYCLES);
+            FlightUtils.waitForFlightExponential(context.getStairway(), flightId,
+                Duration.ofMillis(10), Duration.ofMinutes(5));
```

Comment:
Oooh, 10ms is very short. Does it ever complete the first time through?

Reply:
Probably not. The fastest I've seen this flight go is 36 ms. My requirement from product is for this to be "instantaneous", so I went as low as I figured I could go.

Reply:
Bumping it to 50 ms.